posthog/temporal/data_imports/pipelines/pipeline/utils.py

```diff
@@ -478,6 +478,10 @@ def table_from_py_list(table_data: list[Any], schema: Optional[pa.Schema] = None
     Convert a list of Python dictionaries to a PyArrow Table.
     This is a wrapper around table_from_iterator for backward compatibility.
     """
+    if schema:
+        nullable_fields = [field.with_nullable(True) for field in schema]
+        schema = pa.schema(nullable_fields)
```

Comment:
Yeah we dont really want to just default all fields to be nullable - we get back the types when querying clickhouse and run it through [here](https://github.com/PostHog/posthog/blob/master/posthog/temporal/common/clickhouse.py#L136-L173) to build the schema, but for some reason the nullable types aren't returning properly, so worth stepping through this logic to make sure it's working as we expect it to